src/queries/action-type-queries.ts

```diff
@@ -17,8 +20,10 @@ export type ActionTypeType = {
   [K in keyof ActionTypeRow as SnakeToCamel<K>]: ActionTypeRow[K];
 };
 
-export const findAllUserActions = () => sql<ActionTypeType>`
-  SELECT *
-  FROM action_type
-  WHERE is_user_action IS TRUE;
-`;
+export const findAllUserActions = fetchAny<void, ActionTypeType>(
```

Comment:
Should be `never` instead of `void`.

Reply:
Thanks a lot! Still learning the finer points of TypeScript :)

Reply:
With `void`:
![image](https://user-images.githubusercontent.com/8604342/141689821-2d3a0ca6-d08e-40d9-8a2d-001372534344.png)
With `never`:
![image](https://user-images.githubusercontent.com/8604342/141689847-dba70bb3-03c0-4197-a5fa-2684134ad85c.png)

Suggestions?